webseer.php

```diff
@@ -492,23 +493,22 @@ function clearFilter() {
 			applyFilter();
 		});
 	});
-
 	</script>
 	<?php
 
 	html_start_box(__('Webseer Site Management', 'webseer') , '100%', '', '3', 'center', 'webseer_edit.php?action=edit');
 	?>
 	<tr class='even noprint'>
-		<form id='form_webseer' action='webseer.php'>
-		<input type='hidden' name='search' value='search'>
 		<td class='noprint'>
+		<form id='form_webseer' action='webseer.php'>
+			<input type='hidden' name='search' value='search'>
 			<table class='filterTable'>
 				<tr class='noprint'>
 					<td>
 						<?php print __('Search', 'webseer');?>
 					</td>
 					<td>
-						<input type='text' id='rfilter' size='30' value='<?php print get_request_var('rfilter');?>'>
+						<input type='text' class='ui-state-default ui-corner-all' id='rfilter' size='30' value='<?php print html_escape_request_var('rfilter');?>'>
```

Comment:
Aren't these added by jqueryUI ?  Not sure we should be setting them directly, was there a reason for it?

Reply:
I do not know.  I only know to look for examples in the base code such as data_sources.php (~1426), graphs.php.  I probably copied something out of context.  I looked around but I did not find the proper method.  Where should I look for an example of what should be coded?

Reply:
We can let it pass for now.